system-tests/projects/unify-onboarding/cypress.config.js

```diff
@@ -1,15 +1,13 @@
+const { devServer } = require('@cypress/webpack-dev-server')
+
 module.exports = {
   'component': {
     devServer (cypressDevServerConfig) {
-      const { startDevServer } = require('@cypress/webpack-dev-server')
-
-      return startDevServer({
-        options: cypressDevServerConfig,
-        webpackConfig: {
-          output: {
-            publicPath: '/',
-          },
-        } })
+      return devServer(cypressDevServerConfig, {
```

Comment:
Should we move the `devServerConfig` out of the function into the `component` object, or is it done in the `devServer` function for a reason?